lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -765,6 +767,7 @@ MayaUsdProxyShapeBase::preEvaluation(const MDGContext& context, const MEvaluatio
         evaluationNode.dirtyPlugExists(loadPayloadsAttr) ||
         evaluationNode.dirtyPlugExists(inStageDataAttr)) {
         _IncreaseUsdStageVersion();
+        MayaUsdProxyStageInvalidateNotice(*this).Send();
```

Comment:
Sean I think you just forgot to add this here?

Reply:
Yes I guess. I wasn't aware that what we did in setDependantsDirty needed to have matching here. Under what circumstance would this method be called and not setDependantsDirty?

Reply:
If the EM claims the proxyShape node for parallel evaluation then the proxyShape won't get dirty propagation and setDependentsDirty won't be called. preEvaluate & postEvaluate get called when the node is claimed by the EM, so whatever logic was happening during dirty propagation needs to occur there instead.